site/styles/_share-modal.scss

```diff
@@ -32,7 +36,14 @@
     @extend .position-relative;
     i {
       @extend .position-absolute;
+      font-size: 20px;
       left: -30px;
+
+      /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+      svg {
```

Comment:
This never gets applied to the Letterboxd icon because that svg has inline styles.

Reply:
- removed